test_pawn.py

```diff
@@ -0,0 +1,31 @@
+from game import Game
+
+"""Pawn Moves Test"""
+
+
+def test_indices_to_coordinates():
+    state = Game()
+    i, j = 6, 3
+    result = state.board.indeces_to_coordinates(i, j)
+    print(result)
+    assert result == "E7"
+
+
+def test_forward():
+    state = Game()
+    for col in state.board.columns.keys():
+        position = col + str(7)
+        result = state.board.valid_moves(position)
+        assert len(result) == 2
+        assert col + str(6) in result
+        assert col + str(5) in result
+
+
+def test_diagonal():
```

Comment:
i think we need another test (or maybe combine these tests i would think) to test the functionality of valid_moves together. Because right now, we are testing what a valid piece moves are, but we aren't testing if valid_moves are also returning incorrect valid_moves. Like if the valid_moves function just returned every position on the board, it would pass these tests

Reply:
Checking for the length of the list returned by **valid_moves** is equivalent to checking only the correct values are being returned. Added some complexity to **test_diagonal** and will need to add more tests once a **move** method is created.